src/languages/en.ts

```diff
@@ -3327,6 +3327,72 @@ const translations = {
                 },
             },
         },
+        nsqs: {
+            setup: {
+                title: 'NetSuite setup',
```

Comment:
Let's rename everything that's `NetSuite` to `NSQS` here and below.

Reply:
The default value of imports mapping is `NETSUITE_DEFAULT` which translates to `NetSuite employee default`. Should I:
1. Change default value (requires BE changes) to `NSQS_DEFAULT` and create a new translation for this entity
2. Keep `NETSUITE_DEFAULT` but use custom translation for NSQS
3. Do nothing

<img width="373" alt="Screenshot 2025-02-04 at 1 46 51 PM" src="https://github.com/user-attachments/assets/dbaad615-d623-4891-b54a-04b7f513d587" />